cas-client-core/src/main/java/org/jasig/cas/client/util/CommonUtils.java

```diff
@@ -259,70 +260,59 @@ private static boolean requestIsOnStandardPort(final HttpServletRequest request)
     }
 
     /**
-         * Constructs a service url from the HttpServletRequest or from the given
-         * serviceUrl. Prefers the serviceUrl provided if both a serviceUrl and a
-         * serviceName.
-         *
-         * @param request  the HttpServletRequest
-         * @param response the HttpServletResponse
-         * @param service the configured service url (this will be used if not null)
-         * @param serverNames the server name to  use to constuct the service url if the service param is empty.  Note, prior to CAS Client 3.3, this was a single value.
-         *           As of 3.3, it can be a space-separated value.  We keep it as a single value, but will convert it to an array internally to get the matching value. This keeps backward compatability with anything using this public
-         *           method.
-         * @param artifactParameterName the artifact parameter name to remove (i.e. ticket)
-         * @param encode whether to encode the url or not (i.e. Jsession).    
-         * @return the service url to use.
-         */
+     * Constructs a service url from the HttpServletRequest or from the given
+     * serviceUrl. Prefers the serviceUrl provided if both a serviceUrl and a
+     * serviceName.
+     *
+     * @param request the HttpServletRequest
+     * @param response the HttpServletResponse
+     * @param service the configured service url (this will be used if not null)
+     * @param serverNames the server name to  use to constuct the service url if the service param is empty.  Note, prior to CAS Client 3.3, this was a single value.
+     *           As of 3.3, it can be a space-separated value.  We keep it as a single value, but will convert it to an array internally to get the matching value. This keeps backward compatability with anything using this public
+     *           method.
+     * @param serviceParameterName the service parameter name to remove (i.e. service)
+     * @param artifactParameterName the artifact parameter name to remove (i.e. ticket)
+     * @param encode whether to encode the url or not (i.e. Jsession).
+     * @return the service url to use.
+     */
     public static String constructServiceUrl(final HttpServletRequest request, final HttpServletResponse response,
```

Comment:
we need to confirm this change won't break Spring Security integration.  Some of our CommonUtils are "public API" :-(